k8s/airflow-operator/Makefile

```diff
@@ -0,0 +1,67 @@
+include ../vendor/marketplace-tools/app.Makefile
+include ../vendor/marketplace-tools/crd.Makefile
+include ../vendor/marketplace-tools/gcloud.Makefile
+include ../vendor/marketplace-tools/marketplace.Makefile
+include ../vendor/marketplace-tools/ubbagent.Makefile
+include ../vendor/marketplace-tools/var.Makefile
+
+
+TAG ?= latest
+$(info ---- TAG = $(TAG))
+
+APP_DEPLOYER_IMAGE ?= $(REGISTRY)/airflow-operator/deployer:$(TAG)
+NAME ?= airflow-operator-1
+APP_PARAMETERS ?= { \
+  "name": "$(NAME)", \
+  "namespace": "$(NAMESPACE)" \
+}
+TESTER_IMAGE ?= $(REGISTRY)/airflow-operator/tester:$(TAG)
+APP_TEST_PARAMETERS ?= { \
+  "testerImage": "$(TESTER_IMAGE)" \
+}
+
+
+app/build:: .build/airflow-operator/deployer \
+            .build/airflow-operator/airflow-operator \
+            .build/airflow-operator/tester
+
+
+.build/airflow-operator: | .build
+	mkdir -p "$@"
+
+
+.build/airflow-operator/deployer: deployer/* \
+                                manifest/* \
+                                schema.yaml \
+                                .build/marketplace/deployer/envsubst \
+                                .build/var/APP_DEPLOYER_IMAGE \
+                                .build/var/REGISTRY \
+                                .build/var/TAG \
+                                | .build/airflow-operator
+	$(call print_target, $@)
+	docker build \
+	    --build-arg REGISTRY="$(REGISTRY)/airflow-operator" \
+	    --build-arg TAG="$(TAG)" \
+	    --tag "$(APP_DEPLOYER_IMAGE)" \
+	    -f deployer/Dockerfile \
+	    .
+	docker push "$(APP_DEPLOYER_IMAGE)"
+	@touch "$@"
+
+
+.build/airflow-operator/tester: apptest/**/*
+	$(call print_target, $@)
+	cd apptest/tester \
+	    && docker build --tag "$(TESTER_IMAGE)" .
+	docker push "$(TESTER_IMAGE)"
+	@touch "$@"
+
+
+.build/airflow-operator/airflow-operator: .build/var/REGISTRY \
+                                      .build/var/TAG \
+                                      | .build/airflow-operator
+	$(call print_target, $@)
+	docker pull gcr.io/airflow-operator/airflow-operator:v1alpha1
```

Comment:
Does the operator always install the newest app version? Can we have some kind of a matching between the operator's tag and the Airflow app version (I mean like: `v1alpacha1-1.10` for the operator)? Otherwise we will soon have an incompatibility with the app's tag and the actual version of the Airflow software being installed.

Reply:
The operator is not supposed to be tied to the airflow version. The same operator could be used across multiple versions of airflow. This one recommends 1.9, 1.10 versions of airflow.